internal/log/log_test.go

```diff
@@ -61,3 +62,16 @@ func TestToMode(t *testing.T) {
 		ToMode("invalid")
 	}()
 }
+
+type writerMock struct{}
+
+func (w *writerMock) Write(p []byte) (n int, err error) {
+	return len(p), nil
+}
+
+func TestSetSync(t *testing.T) {
```

Comment:
Unsure about the value of this test tbh...

Reply:
not much value, kind of making codecov happy... which TBH, is a bit annoying